src/main/java/io/takari/aether/okhttp/OkHttpAetherClient.java

```diff
@@ -288,6 +288,13 @@ public String getHeader(String name) {
     public InputStream getInputStream() throws IOException {
       return conn.body().byteStream();
     }
+
+    @Override
+    public void close() {
+      if (conn != null) {
```

Comment:
I don't believe this can be `null` but it is probably a good idea to add a null guard to `ResponseAdapter` constructor